Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+gem 'omniauth-github'
```

Comment:
are you actually going to use this? if you're going to do oauth by hand, you don't need this at all!